MoveCollection.java

```diff
@@ -0,0 +1,116 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class MoveCollection {
+	
+	/** The current number of MOVEs in the array */
+	private int numMoves; 
+	
+	/** The array to contain the MOVEs */
+	private Move moveArray[]; 
+	
+	/** The name of the data file that contains MOVEs-set data */
+	private String sourceName;
+		
+	/** Boolean flag to indicate whether the MOVEs-set collection was
+		    modified since it was last saved. */
+	private boolean modified;
+	
+	public MoveCollection()
+	{
+		numMoves = 0; 
+		moveArray = new Move[7];
+	}
+	
+	// A method to add/modify a move to the array. 
+	public void addOrModifyMove(String name, String type, String category, String damage)
+	{
+		try {	// running time error prevention.   
+				// double the size of the array if the current array is full. 
+			if(numMoves == moveArray.length-1)
```

Comment:
probably won't ever need to resize the array to something bigger than 4, unless you end up deciding to go with the move dictionary that Joe has mentioned in our meetings